### ISSUE TITLE
REFL009 don't warn if null checked

### ISSUE THREAD

**Issue by JohanLarsson** [DotNetAnalyzers/ReflectionAnalyzers#195]
```cs
var property = type.GetProperty("P");
if (property != null)
{
}
```
